addons/sys_zeus/fnc_handleZeusSpeakPressUp.sqf

```diff
@@ -18,6 +18,9 @@
 [{!(GVAR(keyDownWait))}, {
     player setVariable [QGVAR(inZeus), false, true];
 
+    // Stop speaking
```

Comment:
Have a look at how MultiPTT are handled. It may solve your issue: https://github.com/IDI-Systems/acre2/blob/master/addons/sys_core/fnc_handleMultiPttKeyPress.sqf#L22

Otherwise this looks good.

Reply:
Yeah, should be handled the same way, but can be done in separate PR.

Reply:
I would rather not. It should be something handled in this PR

Reply:
Let's improve it then @synixebrett. :D